.circleci/config.yml

```diff
@@ -79,7 +79,7 @@ jobs:
           command: |
               C:\Users\circleci\Miniconda3\shell\condabin\conda-hook.ps1
               conda activate featuretools
-              pytest featuretools\ -n 2
+              pytest featuretools\ -n 2 --ignore=featuretools\dask-tests-tmp\
```

Comment:
Let's add this to the default pytest options in `setup.cfg`.

The reason the linux tests don't currently require this ignore flag is a side effect of the linux tests packaging the library before running the tests.  The folder isn't being included in the packaged library, so it's not necessary to ignore it.  Since the windows tests don't package the library, they do need to ignore the folder.

This can also be a problem when running the tests locally, let's make pytest ignore the folder by default.